internal/abci/abci.go

```diff
@@ -304,6 +297,20 @@ func NewAbciApp(ctx context.Context, cfg *AbciConfig, snapshotter SnapshotModule
 		}
 	}
 
+	migrationStatus, err := migrator.MigrationStatus(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get migration status: %w", err)
+	}
+	app.log.Infof("Migration status: %s", migrationStatus.String())
+	networkParams.MigrationStatus = migrationStatus
+	app.lastMigrationStatus = migrationStatus
```

Comment:
Related to my above comment, you duplicate the variable in both locations here. Why?

Also, I think we should get rid of the log. If we want to know the migration status at each height, we should just add it as a field to some other log that logs the block height.